### ISSUE TITLE
Vis bare tog som går i en bestemt retning

### ISSUE THREAD

**Issue by Chr1stian** [entur/tavla#1533]
For f.eks Drammen stasjon kan man filtrere på bare de togene man ønsker, men det vil fortsatt vise de uansett hvilken retning de går i. Hvis man f.eks bare er interessert i avganger mot Oslo, hadde det vært praktisk om det var mulig å filtrere på det også

**Comment by emilielr** [comment]
Hei, og takk for innspill! Vi skal ta en titt på dette, og legge det i listen over ting vi skal gjøre.

**Comment by tsobuskerudbyen** [comment]
+1 på denne. Kan også være laget universelt, slik at alle kollektivstopp har støtte for retninger i Tavla.

**Comment by emilielr** [comment]
@tsobuskerudbyen Det skal være mulig å filtrere på retning ved å velge de spesifikke plattformene, også filtere på linje. Slik jeg forstår det vil det heller være ønskelig at man kan velge alle tog som går mot Oslo f. eks?

**Comment by Chr1stian** [comment]
Det er i hvert fall akkurat hva jeg ønsker meg med dette issuet. Om @tsobuskerudbyen ønsker noe annet/mer bør det kanskje være ett nytt issue

**Comment by tsobuskerudbyen** [comment]
Ja, ser at det dette ikke er så aktuelt for buss da. Takk for tips.

**Comment by emilielr** [comment]
Hei @Chr1stian @tsobuskerudbyen ! Vi har nå rullet ut en ny funksjon hvor man kan kombinere flere stoppesteder i ett. Dette finner dere under innstillinger for hver tavle, og under "visningstype". Da kan man velge de plattformene + linjene man ønsker, også kombinere alle disse. Gjerne test ut funksjonaliteten, og om det dekker behovet med å kun se avganger i en retning. Vi tar gjerne i mot tilbakemeldinger, eller om det er andre ting dere savner i Tavla.
